napari/_qt/_tests/test_qt_utils.py

```diff
@@ -36,7 +35,6 @@ def err():
     obj.test_signal.connect(err)
     with qt_signals_blocked(obj):
         obj.go()
-        qtbot.wait(750)
```

Comment:
Not sure what this is waiting for. The connection should be executed on the main/test thread.